vignettes/policy-files.Rmd

```diff
@@ -18,7 +18,9 @@ There are two ingredients that are needed to calculate course grades with `grade
 
 Policy files are stored as [.yaml](https://www.cloudbees.com/blog/yaml-tutorial-everything-you-need-get-started) files, a simple format for storing hierarchical data in a text file with minimal markup. Here is an example of a policy file for a course with a fairly simple grading scheme.
 
-```yaml
+Note that category names cannot be the same as any assignment names.
+
+``` yaml
 `r xfun::file_string(system.file("extdata", "simple_policy.yaml", package = "gradebook"))`
```

Comment:
This code chunk will run properly if we change to "policy_demo.yaml", since the demo policy has been renamed for this update from simple_policy.